packages/docs/fluent-editor/demos/flow-chart.vue

```
@@ -0,0 +1,55 @@
+<script setup lang="ts">
+import type FluentEditor from '@opentiny/fluent-editor'
+import LogicFlow from '@logicflow/core'
```

ESLint

[error] 'LogicFlow' is defined but never used.

(unused-imports/no-unused-imports)